### DISCUSSION TITLE
Thrustmaster TMX

### DISCUSSION THREAD

**Discussion by jdvalve** [scarburato/t150_driver#32] [Q&A]
Hi, can this driver be adapted for the TMX wheel? since they are basically the same wheel (so I've read)

**Comment by scarburato** [comment]
idk first of all it needs support for initialization.

you could try to [un-comment this line and increase the size by 1](https://github.com/scarburato/hid-tminit/blob/master/hid-tminit.c#L74) and see if it inits correctly (check if it appears as a TMX when running `lsusb`). you may have to temporary blacklist `hid-thrustmaster`.

**Comment by albfan** [comment]
I think this should be just a proper issue, not a discussion, as there's tentative code commented for that device
